tests/PropertySpec/Exec.hs

```diff
@@ -0,0 +1,47 @@
+module PropertySpec.Exec
+  ( additionSpec
+  , subtractionSpec
+  , multiplicationSpec
+  , divisionSpec
+  )where
+
+import           Data.Foldable                               (fold, foldl1)
+import           Data.List.NonEmpty                          (intersperse)
+import           ExecSpec.Util                               (num)
+import           Graphics.Implicit.ExtOpenScad.Definitions   (OVal (ONum))
+import           Graphics.Implicit.ExtOpenScad.Eval.Constant (runExpr)
+import           HaskellWorks.Hspec.Hedgehog                 (requireProperty)
+import           Hedgehog                                    (diff, forAll)
+import qualified Hedgehog.Gen                                as Gen
+import qualified Hedgehog.Range                              as Range
+import           Prelude                                     (Bool, Floating, String, Double,
+                                                              Ord, fail, show,
+                                                              ($), (&&), (+),
+                                                              (.), (<$>), (<=), (*), (/),
+                                                              (<>), (>=), (-))
+import           Test.Hspec                                  (Spec, it)
+
+approx :: (Floating a, Ord a) => a -> a -> a -> Bool
```

Comment:
this function and the next one deserve a one-liner comment each.

Reply:
I don't think so. The function isn't exported and this pattern of comparisons is prevalent if you are comparing floating point numbers.